packages/server-wallet/src/wallet/index.ts

```diff
@@ -142,6 +147,15 @@ export class Wallet implements WalletInterface, ChainEventSubscriberInterface {
     this.chainService = new MockChainService();
   }
 
+  public async registerAppDefintion(appDefinition: string): Promise<void> {
```

Comment:
Is it worth writing a chain unit test for this function?

Reply:
Good point! Done here: f36ac33